packages/matchbox/src/components/ProgressBar/styles.js

```diff
@@ -0,0 +1,40 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const outerBase = () => `
+  position: relative;
+  background-color: ${tokens.color_gray_300};
+  border-radius: ${tokens.borderRadius_100};
+  width: 100%;
+  margin-bottom: 0.75rem;
```

Comment:
I don't think including a baked-in `margin-bottom` is a good idea - rather, developers should just use the exposed `styled-system` props to position each component instance. You can do the same in the Storybook file!